e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -364,10 +355,20 @@ describe("#39152 sharing an unsaved question", () => {
           });
 
           openSharingMenu("Embed");
-          cy.findByTestId("sharing-pane-public-embed-button").within(() => {
-            cy.findByText("Get an embed link").click();
-            cy.findByTestId("copy-button").realClick();
+
+          modal().findByText("Get embedding code").click();
+
+          // mock clipboardData so that copy-to-clipboard doesn't use window.prompt, pausing the tests
```

Comment:
cypress was getting stuck on CI, locally I was seeing a prompt from copy-to-clipboard which uses window.prompt as fallback if it can't use clipboardData.setData, so I mocked the function and tests passed in ci

I'm not sure why it was working before this pr

Reply:
https://github.com/sudodoki/copy-to-clipboard/blob/113cb357395c3dd623450f8af4d05e66e14c68cb/index.js#L95